### ISSUE TITLE
Enhance contrast: search field placeholder text

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/thanksroy#17]
Placeholder text for full text search is below minimum contrast requirements, clocking in at 3.5:1 (#888888 on #FFFFFF). The contrast must be increased to at least 4.5:1 for text at this size (#767676 works here to meet this requirement).

**Comment by kimisgold** [comment]
The placeholder should have better contrast and use italics to indicate its status not relying on color.
